include/cantera/thermo/HMWSoln.h

```diff
@@ -797,7 +797,7 @@ class HMWSoln : public MolalityVPSSTP
     //! @{
 
     virtual std::string type() const {
-        return "HMWSoln";
+        return "HMW-electrolyte";
```

Comment:
Is there a more descriptive name than `HMW`? While there is copious documentation for the class, the `HMW` doesn't appear to be intuitive.

Reply:
It's the last initials of the three authors who came up with the model. Other sources refer to this model in a similar way as well (e.g. https://academy.gwb.com/high_salinity.php), and I don't know of a more descriptive alternative. We do list the expansion of the acronym in the [glossary](https://cantera.org/documentation/glossary.html#glossary), though that still doesn't provide much information.